### PR TITLE
[9.0] New module 'account_partner_reconcile'

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
Account Partner Reconcile
=========================

This module adds a button "Match Payments" in the customer & suppliers form
view to allow to start the matching of invoices & payments for that partner.

Configuration
=============

The button is visible only to users that belong to the accounting groups
"Accountant" or "Adviser".

### CODE REVIEW

File: account_partner_reconcile/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# © 2017 Eficent Business and IT Consulting Services S.L.
+#        (http://www.eficent.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
```
Comment (LoisRForgeFlow) [inline account_partner_reconcile/models/res_partner.py]: remove 'fields' since it is not used.

**Review by LoisRForgeFlow — COMMENTED** [review]
Very useful one! Just attend travis.

**Review by LoisRForgeFlow — APPROVED** [review]
:+1:

**Comment by manuelfcordoba** [comment]
Great improvement!!! Thanks you... Just tested it and everythigs works fine. :+1:

**Comment by manuelfcordoba** [comment]
Let me know if i can do anything else...

**Comment by gonzaloruzafa** [comment]
:+1:

**Review by gonzaloruzafa — APPROVED** [review]

**Review by jjscarafia — APPROVED** [review]

**Review by MiquelRForgeFlow — APPROVED** [review]
Ready to be merged 🙂

**Pull request merged**